chia/types/blockchain_format/proof_of_space.py

```diff
@@ -85,8 +85,8 @@ def passes_plot_filter(
     return cast(bool, plot_filter[:prefix_bits].uint == 0)
 
 
-def calculate_prefix_bits(constants: ConsensusConstants, height: uint32) -> int:
-    prefix_bits = constants.NUMBER_ZERO_BITS_PLOT_FILTER
+def calculate_prefix_bits(constants: ConsensusConstants, height: uint32) -> uint8:
```

Comment:
this looks like a gratuitous change. I don't see any call sites adjusted in this patch because of this

Reply:
Figured it's ideal to have stricter typing for this, but I guess there's a million other places where this is the case anyways.